### PR TITLE
feat(monitoring): add associated item type icons to list

### PR DESCRIPTION
**Pull request by petrjasek**
SDBELGA-36

### CODE REVIEW

File: scripts/apps/search/components/fields/associatedItems.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {IArticle} from 'superdesk-interfaces/Article';
+
+const TYPES_MAPPING = {
```
Comment (tomaskikutis) [inline scripts/apps/search/components/fields/associatedItems.tsx]: Use clearer naming like `TypeToIconMap`. It's not clear now what does it map to.

File: scripts/apps/search/components/fields/associatedItems.tsx [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
 /**
  * Render icon with count for each item type present in associations.
  */
 export function associatedItems({item}: {item: IArticle}) {
     const associations = Object.values(item.associations || {});
-    const icons = Object.keys(TYPES_MAPPING).map((type) => ({
+    const icons = Object.keys(TYPES_TO_ICONS).map((type) => ({
         type: type,
-        icon: TYPES_MAPPING[type],
+        icon: TYPES_TO_ICONS[type],
```
Comment (tomaskikutis) [inline scripts/apps/search/components/fields/associatedItems.tsx]: Will it work given the entire classname? Only the part after `icon-` used to be passed before.

Comment (petrjasek) [reply scripts/apps/search/components/fields/associatedItems.tsx]: yep the keys didn't change, just values

Comment (tomaskikutis) [reply scripts/apps/search/components/fields/associatedItems.tsx]: Right, I didn't notice that `icon` value is not passed anywhere outside and only used in this file.

**Pull request merged**